examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -89,17 +89,24 @@ else
 fi
 
 
+split_path=(${data_path//\// })
+experiment=${split_path[-1]}
 
+# if [ ! -d $(pwd)/.gt_cache_000000 ]; then
```

Comment:
Do we want to fix this to use the results of the caching build?

Reply:
good point we probably should add this back in conjunction with changing the name of the cache plan